src/base/static/state/ducks/map.js

```diff
@@ -241,7 +248,7 @@ export default function reducer(state = INITIAL_STATE, action) {
           ...state.layerStatuses,
           [action.payload.id]: {
             ...state.layerStatuses[action.payload.id],
-            ...action.payload,
+            loadStatus: action.payload.loadStatus,
```

Comment:
Minor change, related to this comment: https://github.com/mapseed/platform/pull/1086#discussion_r238083453